src/data/screenshots.json

```diff
@@ -10,7 +10,7 @@
         "content": {
             "textblock": [
                 "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 3.2 (available since 19. April, 2023). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
-                "With this update we are fixing bugs in the app. It does not contain any new features.",
+                "With this update we are fixing bugs in the app. It does not contain any new features. The screenshots page displays the implementation prior to the function reduction on May 1, 2023. For more information, please refer to the <a href='../blog/2023-05-02-reduction-of-functions/' target='_blank'>blog</a>.",
```

Comment:
Please fix the date in the previous line (not your error!).

It should say "April 19, 2023" in English format.

Reply:
Resolved with https://github.com/corona-warn-app/cwa-website/commit/f56fa649bc7f1bf9a4605566477be705703b95e3